oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,15 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+
+  $(".dropdown").hover(
+    function() {
+      $('.dropdown-menu', this).not('.in .dropdown-menu').stop(true, true).slideDown("fast");
+      $(this).toggleClass('open');
+    },
+    function() {
```

Comment:
I see somewhere you have used ` ' ' (single quotes)` while at some places you have used ` " " (double quotes)`. Let's just be consistent and use only `single quotes` only.